web/src/message_fetch.js

```diff
@@ -54,7 +53,6 @@ function process_result(data, opts) {
 
     huddle_data.process_loaded_messages(messages);
     recent_view_ui.process_messages(messages);
-    inbox_ui.update();
```

Comment:
So I think we have the `recent_view_ui` call here because fetching messages can change the sort order; I think that's not a problem for the inbox view, because it only sorts by IDs of unread messages, and those can't change when fetching older message history (unless `message_util` found new unreads, in which case that code path will take care of the update). So this seems right.